src/main/java/wdl4s/parser/MemoryUnit.java

```diff
@@ -0,0 +1,22 @@
+package wdl4s.parser;
+
+public enum MemoryUnit {
+    Bytes(1, "B"),
+    KiB(1 << 10, "KiB"),
+    MiB(1 << 20, "MiB"),
+    GiB(1 << 30, "GiB"),
+    TiB(1099511627776.0, "TiB"), // 1 << 40
```

Comment:
Yeah, turns out the compiler doesn't like `1 << 40`.  It interprets it as `1 << 8` because ints are 32 bits.  `1L << 40` actually does produce the right number but it caused the tests to fail for a really weird reason.  I'll investigate.